app/views/new/placement-locations.html

```diff
@@ -5,32 +5,45 @@
 {% block content %}
   <form action="{{ paths.next }}" method="post" class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      {{ govukCheckboxes({
-        idPrefix: code + "-locations",
-        name: code + "-locations",
-        fieldset: {
-          legend: {
-            text: title,
-            isPageHeading: true,
-            classes: "govuk-fieldset__legend--xl"
-          }
-        },
-        hint: {
-          classes: "govuk-!-margin-bottom-6",
-          html: "<a href=\"/locations\">Manage all your locations</a> to add to or edit this list."
-        },
-        items: items
-      }) }}
-
-      {% if change %}
-        {{ govukButton({
-          text: "Save and publish changes"
+      {% if items.length > 0 %}
+        {{ govukCheckboxes({
+          idPrefix: code + "-locations",
+          name: code + "-locations",
+          fieldset: {
+            legend: {
+              text: title,
+              isPageHeading: true,
+              classes: "govuk-fieldset__legend--xl"
+            }
+          },
+          hint: {
+            classes: "govuk-!-margin-bottom-6",
+            html: "<a href=\"/locations\">Manage all your locations</a> to add to or edit this list."
+          },
+          items: items
         }) }}
-        <p class="govuk-body"><a href="{{ paths.back }}">Cancel changes</a></p>
+
+        {% if change %}
+          {{ govukButton({
+            text: "Save and publish changes"
+          }) }}
```

Comment:
Should this just be "Save" as is doesn’t necessarily publish anything (although possibly might, if we decide to do a map or similar for courses where you can't select a school).

```suggestion
          {{ govukButton({
            text: "Save"
          }) }}
```